src/main/java/org/openbase/bco/bcozy/view/InfoPane.java

```diff
@@ -77,62 +91,107 @@ public Label getMouseOverText() {
         return textLabel;
     }
 
-    public static void info(final String identifier) {
-        assert identifier != null;
+    public static InfoPaneConfigurer info(final String identifier) {
+        final String style;
+        if (BCozy.baseColorIsWhite) {
+            style = ("-fx-text-fill: white;");
+        } else {
+            style = ("-fx-text-fill: black;");
+        }
+
+        return show(identifier, style);
+    }
+
+    public static InfoPaneConfigurer confirmation(final String identifier) {
+        return show(identifier, "-fx-text-fill: green;");
+    }
+
+    public static InfoPaneConfigurer warn(final String identifier) {
+        return show(identifier, "-fx-text-fill: orange;");
+    }
+
+    public static InfoPaneConfigurer error(final String identifier) {
+        return show(identifier, "-fx-text-fill: red;");
+    }
+
+    public static InfoPaneConfigurer show(final String identifier, String style) {
+        Objects.requireNonNull(identifier);
+        Objects.requireNonNull(style);
+
         try {
-            final ObserverLabel textLabel = getInstance().textLabel;
+            final InfoPane infoPane = getInstance();
+            infoPane.resetTimeline();
+
             Platform.runLater(() -> {
-                if (BCozy.baseColorIsWhite) {
-                    textLabel.setStyle("-fx-text-fill: white; -fx-font-size: 16;");
-                } else {
-                    textLabel.setStyle("-fx-text-fill: black; -fx-font-size: 16;");
-                }
-                textLabel.setIdentifier(identifier);
+                infoPane.setBackground(null);
+                infoPane.textLabel.setStyle(style + "-fx-font-size: 16;");
+                infoPane.textLabel.setIdentifier(identifier);
             });
+            return new InfoPaneConfigurer();
         } catch (CouldNotPerformException ex) {
             ExceptionPrinter.printHistory("Could not print user feedback!", ex, LOGGER);
         }
+        return null;
     }
 
-    public static void confirmation(final String identifier) {
-        assert identifier != null;
+    /**
+     * Hides the InfoPane.
+     */
+    public static void hide() {
         try {
-            final ObserverLabel textLabel = getInstance().textLabel;
-            Platform.runLater(() -> {
-                textLabel.setStyle("-fx-text-fill: green; -fx-font-size: 16;");
-                textLabel.setIdentifier(identifier);
-            });
+            show("", "");
+            getInstance().setBackground(null);
+            getInstance().resetTimeline();
         } catch (CouldNotPerformException ex) {
             ExceptionPrinter.printHistory("Could not print user feedback!", ex, LOGGER);
         }
     }
 
-    public static void warn(final String identifier) {
```

Comment:
API brake: why did you remove this method?

Reply:
This method was not removed. The return type changed to InfoPaneConfigurer but all previous calls still work the same way they did before. So I would not consider this an API break?